RFC/src/RFC-0120_Consensus.md

```diff
@@ -140,13 +144,40 @@ The timestamp MUST confirm to the following:
 
 #### Output_mr
 
-This is the merkle root of the outputs. This is calculated in the following way: Hash (txo MMR root  || roaring bitmap hash of UTXO indices).
+This is the merkle root of the outputs. It MUST be calculated in the following way: Hash (`txo MMR root`  || 
+Hash(`roaring bitmap`)). The output_mr field is used to represent the entire UTXO set in the blockchain; this is a 
+proof that every UTXO exists and that a TXO is either spent or unspent. The `txo MMR root` MUST be the merkle 
+mountain range (MMR) root of all transactional outputs in existence. The `roaring bitmap hash` MUST be a roaring bitmap 
+of every spent transactional output (aka every input in a block). The index used in the roaring bitmap represents the 
+leaf index of the UTXO in the output MMR. 
 
 The output_mr MUST confirm to the following:
 
 * Represented as an array of unsigned 8-bit integers (bytes) in little-endian format.
 * The hashing function used MUST be blake2b with a 256 bit digest.
 
+#### Output_mmr_size
+
+This is the total size of the leaves in the output merkle mountain range.
+
+The Output_mmr_size MUST confirm to the following:
+
+* Represented as a single unsigned 64-bit integer.
+
+#### Witness_mr
```

Comment:
I think this should just be called "Input_mr" and should only include a hash of the input data, and should include all of the fields of the input, so that a proof can be constructed.
There is no need to include the output data because it is already covered by the output_mr and the range proof mr. 

It also should be mentioned that it is a simple merkle tree for this block only and does not need to include previous block's data like the output and kernel mmr's, since it is only used to verify the data in this block has not changed